awsiot/mqtt_connection_builder.py

```diff
@@ -90,6 +90,8 @@
 
     **enable_metrics_collection** (`bool`): Whether to send the SDK version number in the CONNECT packet.
         Default is True.
+
+    **http_proxy_options** (:class: 'awscrt.http.HttpProxyOptions'): http proxy options to use
 """
 
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Update the docs for `def websockets_with_custom_handshake` and `def websockets_with_default_aws_signing` to say that `websocket_proxy_options` are deprecated, like:
~~websocket_proxy_options (awscrt.http.HttpProxyOptions): If specified, a proxy is used when connecting.~~
```
        websocket_proxy_options (awscrt.http.HttpProxyOptions): Deprecated,
            for proxy settings use `http_proxy_options` (described in
            :mod:`common arguments<awsiot.mqtt_connection_builder>`)
```